app/src/utils/render-string-template.ts

```diff
@@ -67,15 +67,15 @@ export function renderDisplayStringTemplate(
 		set(fieldsUsed, key, fieldsStore.getField(collection, key));
 	}
 
-	const { aliasFields } = useAliasFields(ref(fields));
+	const { aliasedFields } = useAliasFields(ref(fields), ref(collection));
 
 	const parsedItem: Record<string, any> = {};
 
 	for (const key of fields) {
 		const value =
-			!aliasFields.value?.[key] || get(item, key) !== undefined
+			!aliasedFields.value?.[key]?.fields[0] || get(item, key) !== undefined
 				? get(item, key)
-				: get(item, aliasFields.value[key].fullAlias);
+				: get(item, aliasedFields.value[key].fields[0]);
```

Comment:
Not 100% sure about this implementation but it looks like `fields: string[]` is the replacement for `fullAlias` and it can only have 1 item but do correct me if i am wrong @Nitwel